web/text_layer_builder.js

```diff
@@ -126,6 +176,7 @@ class TextLayerBuilder {
     }
     this.highlighter?.disable();
     this.accessibilityManager?.disable();
+    this.renderingDone = false;
```

Comment:
Why do we need this, since when `cancel` is called the (current) assumption in the code ought to be that the `TextLayerBuilder`-instance is going away?